src/variables/utils/variableTableDescription.ts

```diff
@@ -0,0 +1,114 @@
+import { h, VirtualDOM, VirtualElement } from '@phosphor/virtualdom';
```

Comment:
Are we phasing out the use of `@phosphor/virtualdom` and instead opting to use React?

Reply:
It depends. Actually, I just start creating a simple layout. Do you think we should stick in use  @phosphor/virtualdom?

Reply:
Ah yes, if you're doing this with a virtual DOM then you may as well use React in a `.tsx` file! If you want to make Phosphor `Widget`, `Panel` etc. instances, then that's cool, too. But a future version of Phosphor won't have `@phosphor/virtualdom`.

Reply:
Hmmm... I didnt know about that. I will fix this.

Reply:
@KsavinN we put together [a guide on using React in JupyterLab](https://jupyterlab.readthedocs.io/en/stable/developer/virtualdom.html). Feel free to ping me if you have any Qs about it?

Reply:
Thanks, Saul. I didn't know we had this.